src/main/scala/com/advancedtelematic/campaigner/data/DataType.scala

```diff
@@ -20,7 +20,9 @@ object DataType {
 
   object MetadataType extends Enumeration {
     type MetadataType = Value
-    val install = Value
+    val description = Value
+    val estimatedInstallationDuration = Value
+    val estimatedPreparationDuration = Value
   }
```

Comment:
This can be all in one line `description, estimatedInstallationDuration, estimatedPreparationDuration = Value`

The migration will fail if there are already metadata for some campaigns with `install`.

 I think the capitalization is a bit weird maybe should be `EstimatedInstallationDuration`, or `ESTIMATED_INSTALL_DURATION` ?

Reply:
Capitalization for enums seems to follow that style (at least in campaigner).

Reply:
+1 for capitalization